ql/src/java/org/apache/hadoop/hive/ql/parse/HiveTableName.java

```diff
@@ -106,6 +106,18 @@ public static TableName ofNullable(String dbTableName, String defaultDb) throws
     }
   }
 
+  public static TableName ofNullable(String dbTableName, String defaultDb, String branchName) throws SemanticException {
```

Comment:
why don't you reuse new signature in `ofNullable(String dbTableName, String defaultDb)`